portal/src/OnboardingSurveyLayout.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useContext, useMemo } from "react";
+import { useTheme, Text } from "@fluentui/react";
+import { Context } from "@oursky/react-messageformat";
+import styles from "./OnboardingSurveyLayout.module.css";
+import authgearLogoURL from "./images/authgear_logo_color.svg";
+
+interface LogoProps {}
+
+const Logo: React.VFC<LogoProps> = (_props: LogoProps) => {
+  const { renderToString } = useContext(Context);
+  const theme = useTheme();
+  const logoStyles = useMemo(() => {
+    return {
+      fill: theme.semanticColors.bodyText,
+    };
+  }, [theme]);
+  return (
+    <img
+      style={logoStyles}
+      className={styles.logo}
+      alt={renderToString("system.name")}
+      src={authgearLogoURL}
+    />
+  );
+};
+
+export interface SurveyTitleProps {
+  children?: React.ReactNode;
+}
+
+export function SurveyTitle(props: SurveyTitleProps): React.ReactElement {
+  const theme = useTheme();
+  const styles = useMemo(() => {
+    return {
+      root: {
+        "font-size": "x-large",
+        "font-weight": 600,
+        "white-space": "pre-line",
+        "text-align": "center",
+        color: theme.semanticColors.bodyText,
+        "margin-bottom": "20px",
+      },
+    };
+  }, [theme]);
+  return (
+    <Text styles={styles} variant="large" block={true}>
+      {props.children}
+    </Text>
+  );
+}
+
+export interface SurveySubtitleProps {
+  children?: React.ReactNode;
+}
+
+export function SurveySubtitle(props: SurveySubtitleProps): React.ReactElement {
+  const theme = useTheme();
+  const styles = useMemo(() => {
+    return {
+      root: {
+        "font-size": "medium",
+        "font-weight": 400,
+        "white-space": "pre-line",
+        "text-align": "center",
+        color: theme.semanticColors.bodySubtext,
+      },
+    };
+  }, [theme]);
+  return (
+    <Text styles={styles} variant="small" block={true}>
+      {props.children}
+    </Text>
+  );
+}
+
+export interface SurveyLayoutProps {
+  title: string;
```

Comment:
It should be ReactNode

Reply:
After a bit of thinking, I think this is better:
- if its reactnode, i would have to make SurveyTitle and SurveySubtitle in OnboardingSurveyScreen.tsx instead of OnboardingSurveyLayout.tsx; it would make sense if different pages have different titles and subtitles elements, but this is not the case.
- the nextbutton and backbuttons are reactnode because the `onClick`, `disabled` and `text` needs to be supplied; also the nextbutton is different in the last page (finish suvey) so it makes sense for them to be ReactNodes